urllib3/util/retry.py

```diff
@@ -194,12 +194,20 @@ def get_backoff_time(self):
         backoff_value = self.backoff_factor * (2 ** (consecutive_errors_len - 1))
         return min(self.BACKOFF_MAX, backoff_value)
 
-    def sleep(self):
+    def sleep(self, response=None):
         """ Sleep between retry attempts using an exponential backoff.
```

Comment:
This docstring needs updating (now that I look at it). As it is currently written it implies it only uses an exponential back off. It should be updated to use a more vague title, e.g.,

```py
Sleep between retry attempts.

This method will respect a server's ``Retry-After`` response header and sleep the duration of the time requested. If that is not present, it will use an exponential backoff. By default, the backoff factor is 0 and this method will return immediately.
"""